src/conditional-flow/string-conditions.js

```diff
@@ -1,41 +1,43 @@
 // This function should return true if the passed string is equal to "Hello"
-function isHello (val1) {
-
-  // TODO: write code in this function body to pass the tests
-
+function isHello(val1) {
+  return val1 === "Hello" ? true : false;
 }
 
 // This function should return true if the passed string is not equal to "Hello"
-function isNotHello (val1) {
-
-  // TODO: write code in this function body to pass the tests
-
+function isNotHello(val1) {
+  return !isHello(val1) ? true : false;
```

Comment:
Great reusing the other functions. However you don't need the ternary if here, as the other function returns a boolean value. So you could have just left it as `return !isHello(var1)`